src/Morphir/Visual/Components/FieldList.elm

```diff
@@ -15,7 +15,7 @@ view fields =
         , spacingXY 0 5
         ]
         { columns =
-            [ { header = none
+            [ { header = text ""
```

Comment:
This is necessary because of a strange bug in elm-ui, where if a table header is set to none, the order of the table elements gets reversed in HTML